HDPS/src/LinkedData.cpp

```diff
@@ -17,11 +18,10 @@ SelectionMap::SelectionMap(Type type /*= Indexed*/) :
 }
 
 SelectionMap::SelectionMap(const QSize& sourceImageSize, const QSize& targetImageSize) :
-    Serializable("SelectionMapping"),
-    _type(Type::ImagePyramid),
-    _sourceImageSize(sourceImageSize),
-    _targetImageSize(targetImageSize)
+    SelectionMap(Type::ImagePyramid)
 {
+    _sourceImageSize = sourceImageSize;
+    _targetImageSize = targetImageSize;
```

Comment:
I want to keep the initializations for the `_sourceImageSize` and `_targetImageSize` in the constructor initialization list as before. I see no reason why it should be in the constructor body?